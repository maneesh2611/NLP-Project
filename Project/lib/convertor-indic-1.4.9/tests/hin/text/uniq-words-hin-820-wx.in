-
,
.
(
)
133
1528
1565
17
1826
1950
1996
20
21
731
75
aba
aBiReka
AcCAxiwa
addA
Ae
AegIM
agale
agara
AgarA
Age
ahiMsA
akabara
aKAdZe
Akara
AkAra
AkarRaNa
AkarRiwa
Akqwi
alaga
alaga-alaga
alagAra
alagArajI
alaga-sI
amara
amaranAWa
amarawA
amqwa
amqwamaMWana
aMwararARtrIya
Ane
aneka
anna-jala
anuBava
anuBUwi
anupama
anusAra
anUTe
anUTI
anya
Apa
ApakI
Apako
apane
apanI
ApaxA
AraMBa
Arkatika
arWa
ASaMkA
aSoka
avaSeRa
AvaSyaka
avawAroM
avismaraNIya
AwA
AwaMkavAxa
aWavA
Awe
Awe-Awe
AwI
awIwa
awyaMwa
Axawa
AyA
AyawAkAra
bAbA
bAbara
baca
bacce-bacce
baccoM
badZA
baDZa
baDZAvA
baDZawe
baDZawI
badZe
badZe-badZe
bAga
bAGa
BAga
BagavAna
bAhara
bahawI
bahAxura
bahula
bahuwa
bAja
Bajana
bajawI
Bakwa
balki
baMgAla
baMXana
baMxarapucCa
banA
banAe
banAI
banavAe
banavAyA
banawA
banAyA
bane
BAnuprawApasiMha
bAra
Bara
BArawa
bAre
barPa
basA
bAvajUxa
Bavana
Bavya
bawAI
bawAyA
bAxa
bAxaSAha
bayAna
bAzXa
bazXI
behaxa
bejodZa
bEla
bElagAzva
bETakara
BI
bIca
BIdZa
BIdZa-BAdZa
BikRu
bilkula
binA
bIra
BIwara
BolenAWa
borda
bOxXa
bOYtanikala
brahmakuMda
'brahmakuMda'
brAigana
BUrIsiMha
bUtA
buxXa
bUzxeM
caDZAve
CadZI
cahalakaxamI
cAhawe
cala
calA
cAlaka
calawA
cambA
cAra
caravAhe
cAraxIvArI
cAroM
CatA
cawurBuja
cAzxavAdZI
cennaI
CipawA
ciwriwa
cObIsoM
CodZa
CodZA
cOkI
cola
CotA
cOxahavIM
Dazke
eka
ekAMwa
EsA
evaM
gaDZavAla
gAdZI
gae
gaI
gaMgA
GaMte
GaMtiyAz
gArdana
garmI
GAta
GAtI
gayA
gAzva
GirA
girI
gleSiyaroM
golPa
gOmuKa
gopura
gopurama
GoRiwa
grahaNa
gujara
guPA
gUzjawe
hAive
hajAra
hajAroM
hama
hara
hari
harixvAra
'harixvAra'
havA
havAI
havelI
hAWa
haWiyAra
hE
hEM
hewu
hI
himAlaya
himaliMga
hIroM
ho
hokara
hoM
honA
hone
hotala
howA
howe
howI
huA
hue
hugalI
huI
I
icCA
imArawoM
ina
inakAra
inake
inase
isa
isakA
isake
isakI
isalie
isase
ise
iwihAsa
jA
jaba
jabaki
jAe
jagahoM
jahAja
jahAM
jahAz
jahAzgIra
jAI
jala
jalasrowa
jaMgala
jamIna
jAnA
jAne
jAnI
jarUra
jAwA
jAwe
jAwI
jEsalamera
JIla
jinameM
jinheM
jisa
jisase
jIvana
jIvana-maraNa
jIvikA
jo
joki
jubAna
JUlA
julUsa
jUna
jUxeva
kA
kaBAra
kaBI
kaBI-kaBI
KadZe
kahA
kahAnI
kahawe
kahIM
kaI
kalA
kala-kala
kalakawwA
kalAoM
KAlI
kAliMxa
kAma
kamala
kAPI
kara
KarAbI
kAraNa
karane
karavAI
karavAwe
karavAwI
karavAyA
karawA
karAwA
karawe
karawI
kareMge
karmiyoM
kasbe
kaSmIra
kaWA
ke
KeloM
keMxra
kenxra
kEPe
kevala
ki
kI
KilAnA
KilAyA
kiMvaxaMwiyAM
kiMvaxaMwiyoM
kiMwu
kinAre
kinAroM
kisI
kiwanA
kiyA
ko
koI
KojA
kqRNa
kramAMka
kRewra
KUbasUrawa
kuCa
Kule
kuMBA
lAe
lagA
lagaBaga
lagawA
lagAwAra
lage
lakadZI
lAkara
lakRmaNa
le
leKA-joKA
lekara
lekina
lewA
lie
lIlA
liMga
liyA
liye
loga
logoM
lOta
maCaliyoM
magara
mahala
mahAna
mahArAjA
mahArANA
majabUwa
makAna
malika
maMjila
maMjilA
maMxira
maMxiroM
mana
mAnA
mAnakara
mAnanA
manorama
manwrAlaya
mArga
mAwA
maxrAsa
maXukara
maxurE
maXyakAlIna
mAz
mehanawa
meM
merItAima
mExAnI
mExAnoM
milaka
milawA
milI
mInAkRI
mIrA
moha
mOjUxa
mOwa
mubAraka
mugala
mugXa
muhEyA
muKa
mumawAja
murIxa
mUrwi
muslima
myUjiyama
na
nagara
nahIM
naI-naI
najara
najAroM
najaxIka
nakkASI
nAma
nAmaka
naMxI
narmaxA
nawamaswaka
naxI
ne
nihAranA
nihArane
nikala
nikalane
nikalI
nikAlI
niraMwara
nirmala
nirmANa
nirmiwa
nivAsa
nO
nOkA
nOkAyAna
ora
Ora
oraCA
oralEMdoM
padZA
padZe
pahAdZI
pahAdZoM
pahalagAma
pahale
pahalI
pahaluoM
pahuzca
pahuzcawe
pahuzce
pAMdicerI
paMdiwoM
pAnI
panxraha
para
parvawa
parvawIya
paryataka
paryatakoM
paryatana
pAsa
pAsaMga
paScAwa
paScima
pAvana
paviwra
paviwrawA
pawA
Pawehapura
pawnI
pawWara
paxama
pAzca
pEgodA
pEroteta
pExA
pExala
pI
piCale
Pira
pOdZI
POja
pOXe
POYra
pqRTaBUmi
prakqwi
prAkqwika
praKyAwa
pramANa
pramota
praPulliwa
praSikRiwa
prasixXa
pravAhiwa
prawimA
prawIwa
prawixina
prawyeka
praXAnamaMwrI
prema
PreMca
preraNA
PrI
prOxyogikI
pUjA
pula
PUla
puMCa
puNya
pUrA
purAnI
puRti
pUrvajoM
Puta
pyAlA
PyUrdo
racanA
rahA
rahakara
rahasya
rahawA
rahawe
rahawI
rahe
rahegA
rahI
rAjA
rAjAoM
rAjasWAna
rAjaXAnI
raKA
raKakara
raKavAlI
raKe
rakRA
rAma
rANA
rAnI
rASi
rAswe
regiswAna
rojagAra
romAncakArI
ropa
rUbarU
ruKa
rUmAniyA
rUpa
sa
sA
saBI
SAha
SAhajahAz
Sahara
sAiMsa
sakawe
sakawI
sakegeM
sAkRAwkAra
sAla
salIma
sAmane
samaya
samBAvanAez
SaMBu
saMcAlana
sAMgA
saMgIwa
saMgrahAlaya
SAmila
saMKyA
saMpanna
samrAta
saMsWAnoM
SAMwa
SAMwi
saMxeSa
SAnaxAra
SAnwa
sArABAI
sarakAra
sAre
sarkala
sAwa
sAWa
SawAbxI
sAWa-sAWa
sawI
sAXana
saxiyoM
sAXu
sazBAlakara
sazvArA
se
seMtara
sena
senA
SeRanAga
sI
SiKara
sIkarI
siKAyA
sIlona
Silpa
sImA
siMha
sire
SiRyoM
Siva
SivajI
skOYtalEMda
slAidsa
smAraka
snAna
sOBAgya
sOMxaryayukwa
spesa
SrAina
SraMga
SraxXAlu
SraxXAluoM
SrI
SrIgaNeSa
stadIja
subaha
sUcanA
sUKa
sukUna
suMxara
suMxarawA
sunAI
sunxarawA
surakRA
suramya
sUrya
suSoBiwa
suviXA
svacCa
svarga
svarUpa
swaMBa
sWAna
sWAnIya
sWApawya
sWiwa
swUpa
swUpoM
tApU
TIka
trasta
uBara
udZAna
udZanaKatolA
UMcI
unake
unheM
unhoMne
Upara
usa
usake
usakI
usane
use
usI
uwarakara
uwwara
uwwara-paScima
uwwarAyaNa
UzcA
va
vaha
vAhana
vAhanoM
vahAz
vahIM
vajaha
vAlA
vAle
vAlI
vanaspawi
vandara
vApasa
varksa
varRA
varRaBara
varRoM
varwamAna
vAsiyoM
vAswavika
vAwAvaraNa
vAxI
vAxiyAz
vAxiyoM
ve
vEgA
vEjFAnika
vESvika
viciwra
vijFAna
vijita
vikrama
vIM
viRa
virAjiwa
viRNu
viSeRa
viSeRaNa
viSva
vivAha
vixeSa
vixeSI
vOYtara
vyakwi
vyavasAya
vyavasWA
WA
waba
wAjA
wAjamahala
waka
wakanIkI
wAlAba
wapawA
waraha
waraPa
wata
waWA
wawkAlIna
We
WI
WIM
wiruvanaMwapurama
wIsarI
wo
woraNa
wrayoxaSI
wUPAnoM
xakRiNa
xAnA
XArA
XarawI
Xarma
XarmasWala
XArmika
XArmikawA
Xarohara
xarpa-Silpa
xarSana
xasa
xaSanAmI
xe
xeKa
xeKane
xeKa-reKa
xenA
xene
xewA
xewI
xI
xie
xiKAne
xiKawA
xilAwe
xina
xiSAoM
xiyA
xo
xonoM
xUra
xurGatanA
xUsare
xUsarI
Xvani
xvAra
xvArA
xvIpoM
XyAna
yA
yaha
yahAM
yahAz
yahIM
yAwrA
yAwrI
yAwriyoM
yAxa
yaxi
ye
yuga
yUnesko
