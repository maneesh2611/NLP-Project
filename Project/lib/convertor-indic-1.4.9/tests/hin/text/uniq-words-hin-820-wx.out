-
,
।
(
)
133
1528
1565
17
1826
1950
1996
20
21
731
75
अब
अभिषेक
आच्छादित
अड्डा
आए
आएगीं
अगले
अगर
आगरा
आगे
अहिंसा
अकबर
अखाड़े
आकर
आकार
आकर्षण
आकर्षित
आकृति
अलग
अलग-अलग
अलगार
अलगारजी
अलग-सी
अमर
अमरनाथ
अमरता
अमृत
अमृतमंथन
अंतरराष्ट्रीय
आने
अनेक
अन्न-जल
अनुभव
अनुभूति
अनुपम
अनुसार
अनूठे
अनूठी
अन्य
आप
आपकी
आपको
अपने
अपनी
आपदा
आरंभ
आर्कटिक
अर्थ
आशंका
अशोक
अवशेष
आवश्यक
अवतारों
अविस्मरणीय
आता
आतंकवाद
अथवा
आते
आते-आते
आती
अतीत
अत्यंत
आदत
आया
आयताकार
बाबा
बाबर
बच
बच्चे-बच्चे
बच्चों
बड़ा
बढ़
बढ़ावा
बढ़ते
बढ़ती
बड़े
बड़े-बड़े
बाग
बाघ
भाग
भगवान
बाहर
बहती
बहादुर
बहुल
बहुत
बाज
भजन
बजती
भक्त
बल्कि
बंगाल
बंधन
बंदरपुच्छ
बना
बनाए
बनाई
बनवाए
बनवाया
बनता
बनाया
बने
भानुप्रतापसिंह
बार
भर
भारत
बारे
बर्फ
बसा
बावजूद
भवन
भव्य
बताई
बताया
बाद
बादशाह
बयान
बाँध
बँधी
बेहद
बेजोड़
बैल
बैलगाँव
बैठकर
भी
बीच
भीड़
भीड़-भाड़
भिक्षु
बिल्कुल
बिना
बीर
भीतर
भोलेनाथ
बोर्ड
बौद्ध
बॉटनिकल
ब्रह्मकुंड
'ब्रह्मकुंड'
ब्राइगन
भूरीसिंह
बूटा
बुद्ध
बूँदें
चढ़ावे
छड़ी
चहलकदमी
चाहते
चल
चला
चालक
चलता
चम्बा
चार
चरवाहे
चारदीवारी
चारों
छटा
चतुर्भुज
चाँदवाड़ी
चेन्नई
छिपता
चित्रित
चौबीसों
छोड़
छोड़ा
चौकी
चोल
छोटा
चौदहवीं
ढँके
एक
एकांत
ऐसा
एवं
गढ़वाल
गाड़ी
गए
गई
गंगा
घंटे
घंटियाँ
गार्डन
गर्मी
घाट
घाटी
गया
गाँव
घिरा
गिरी
ग्लेशियरों
गोल्फ
गौमुख
गोपुर
गोपुरम
घोषित
ग्रहण
गुजर
गुफा
गूँजते
हाइवे
हजार
हजारों
हम
हर
हरि
हरिद्वार
'हरिद्वार'
हवा
हवाई
हवेली
हाथ
हथियार
है
हैं
हेतु
ही
हिमालय
हिमलिंग
हीरों
हो
होकर
हों
होना
होने
होटल
होता
होते
होती
हुआ
हुए
हुगली
हुई
ई
इच्छा
इमारतों
इन
इनकार
इनके
इनसे
इस
इसका
इसके
इसकी
इसलिए
इससे
इसे
इतिहास
जा
जब
जबकि
जाए
जगहों
जहाज
जहां
जहाँ
जहाँगीर
जाई
जल
जलस्रोत
जंगल
जमीन
जाना
जाने
जानी
जरूर
जाता
जाते
जाती
जैसलमेर
झील
जिनमें
जिन्हें
जिस
जिससे
जीवन
जीवन-मरण
जीविका
जो
जोकि
जुबान
झूला
जुलूस
जून
जूदेव
का
कभार
कभी
कभी-कभी
खड़े
कहा
कहानी
कहते
कहीं
कई
कला
कल-कल
कलकत्ता
कलाओं
खाली
कालिंद
काम
कमल
काफी
कर
खराबी
कारण
करने
करवाई
करवाते
करवाती
करवाया
करता
कराता
करते
करती
करेंगे
कर्मियों
कस्बे
कश्मीर
कथा
के
खेलों
केंद्र
केन्द्र
कैफे
केवल
कि
की
खिलाना
खिलाया
किंवदंतियां
किंवदंतियों
किंतु
किनारे
किनारों
किसी
कितना
किया
को
कोई
खोजा
कृष्ण
क्रमांक
क्षेत्र
खूबसूरत
कुछ
खुले
कुंभा
लाए
लगा
लगभग
लगता
लगातार
लगे
लकड़ी
लाकर
लक्ष्मण
ले
लेखा-जोखा
लेकर
लेकिन
लेता
लिए
लीला
लिंग
लिया
लिये
लोग
लोगों
लौट
मछलियों
मगर
महल
महान
महाराजा
महाराणा
मजबूत
मकान
मलिक
मंजिल
मंजिला
मंदिर
मंदिरों
मन
माना
मानकर
मानना
मनोरम
मन्त्रालय
मार्ग
माता
मद्रास
मधुकर
मदुरै
मध्यकालीन
माँ
मेहनत
में
मेरीटाइम
मैदानी
मैदानों
मिलक
मिलता
मिली
मीनाक्षी
मीरा
मोह
मौजूद
मौत
मुबारक
मुगल
मुग्ध
मुहैया
मुख
मुमताज
मुरीद
मूर्ति
मुस्लिम
म्यूजियम
न
नगर
नहीं
नई-नई
नजर
नजारों
नजदीक
नक्काशी
नाम
नामक
नंदी
नर्मदा
नतमस्तक
नदी
ने
निहारना
निहारने
निकल
निकलने
निकली
निकाली
निरंतर
निर्मल
निर्माण
निर्मित
निवास
नौ
नौका
नौकायान
ओर
और
ओरछा
ओरलैंडों
पड़ा
पड़े
पहाड़ी
पहाड़ों
पहलगाम
पहले
पहली
पहलुओं
पहुँच
पहुँचते
पहुँचे
पांडिचेरी
पंडितों
पानी
पन्द्रह
पर
पर्वत
पर्वतीय
पर्यटक
पर्यटकों
पर्यटन
पास
पासंग
पश्चात
पश्चिम
पावन
पवित्र
पवित्रता
पता
फतेहपुर
पत्नी
पत्थर
पदम
पाँच
पैगोडा
पैरोटेट
पैदा
पैदल
पी
पिछले
फिर
पौड़ी
फौज
पौधे
फॉर
पृष्ठभूमि
प्रकृति
प्राकृतिक
प्रख्यात
प्रमाण
प्रमोट
प्रफुल्लित
प्रशिक्षित
प्रसिद्ध
प्रवाहित
प्रतिमा
प्रतीत
प्रतिदिन
प्रत्येक
प्रधानमंत्री
प्रेम
फ्रेंच
प्रेरणा
फ्री
प्रौद्योगिकी
पूजा
पुल
फूल
पुंछ
पुण्य
पूरा
पुरानी
पुष्टि
पूर्वजों
फुट
प्याला
फ्यूर्डो
रचना
रहा
रहकर
रहस्य
रहता
रहते
रहती
रहे
रहेगा
रही
राजा
राजाओं
राजस्थान
राजधानी
रखा
रखकर
रखवाली
रखे
रक्षा
राम
राणा
रानी
राशि
रास्ते
रेगिस्तान
रोजगार
रोमान्चकारी
रोप
रूबरू
रुख
रूमानिया
रूप
स
सा
सभी
शाह
शाहजहाँ
शहर
साइंस
सकते
सकती
सकेगें
साक्षात्कार
साल
सलीम
सामने
समय
सम्भावनाएँ
शंभु
संचालन
सांगा
संगीत
संग्रहालय
शामिल
संख्या
संपन्न
सम्राट
संस्थानों
शांत
शांति
संदेश
शानदार
शान्त
साराभाई
सरकार
सारे
सर्कल
सात
साथ
शताब्दी
साथ-साथ
सती
साधन
सदियों
साधु
सँभालकर
सँवारा
से
सेंटर
सेन
सेना
शेषनाग
सी
शिखर
सीकरी
सिखाया
सीलोन
शिल्प
सीमा
सिंह
सिरे
शिष्यों
शिव
शिवजी
स्कॉटलैंड
स्लाइड्स
स्मारक
स्नान
सौभाग्य
सौंदर्ययुक्त
स्पेस
श्राइन
श्रंग
श्रद्धालु
श्रद्धालुओं
श्री
श्रीगणेश
स्टडीज
सुबह
सूचना
सूख
सुकून
सुंदर
सुंदरता
सुनाई
सुन्दरता
सुरक्षा
सुरम्य
सूर्य
सुशोभित
सुविधा
स्वच्छ
स्वर्ग
स्वरूप
स्तंभ
स्थान
स्थानीय
स्थापत्य
स्थित
स्तूप
स्तूपों
टापू
ठीक
ट्रस्ट
उभर
उड़ान
उड़नखटोला
ऊंची
उनके
उन्हें
उन्होंने
ऊपर
उस
उसके
उसकी
उसने
उसे
उसी
उतरकर
उत्तर
उत्तर-पश्चिम
उत्तरायण
ऊँचा
व
वह
वाहन
वाहनों
वहाँ
वहीं
वजह
वाला
वाले
वाली
वनस्पति
वन्डर
वापस
वर्क्स
वर्षा
वर्षभर
वर्षों
वर्तमान
वासियों
वास्तविक
वातावरण
वादी
वादियाँ
वादियों
वे
वैगा
वैज्ञानिक
वैश्विक
विचित्र
विज्ञान
विजिट
विक्रम
वीं
विष
विराजित
विष्णु
विशेष
विशेषण
विश्व
विवाह
विदेश
विदेशी
वॉटर
व्यक्ति
व्यवसाय
व्यवस्था
था
तब
ताजा
ताजमहल
तक
तकनीकी
तालाब
तपता
तरह
तरफ
तट
तथा
तत्कालीन
थे
थी
थीं
तिरुवनंतपुरम
तीसरी
तो
तोरण
त्रयोदशी
तूफानों
दक्षिण
दाना
धारा
धरती
धर्म
धर्मस्थल
धार्मिक
धार्मिकता
धरोहर
दर्प-शिल्प
दर्शन
दस
दशनामी
दे
देख
देखने
देख-रेख
देना
देने
देता
देती
दी
दिए
दिखाने
दिखता
दिलाते
दिन
दिशाओं
दिया
दो
दोनों
दूर
दुर्घटना
दूसरे
दूसरी
ध्वनि
द्वार
द्वारा
द्वीपों
ध्यान
या
यह
यहां
यहाँ
यहीं
यात्रा
यात्री
यात्रियों
याद
यदि
ये
युग
यूनेस्को
